python/src/spark_rapids_ml/classification.py

```diff
@@ -952,8 +971,17 @@ def _logistic_regression_fit(
                 concated = _concat_and_free(X_list, order=array_order)
                 concated_y = _concat_and_free(y_list, order=array_order)
 
+            is_sparse = isinstance(concated, scipy.sparse.csr_matrix) or isinstance(
```

Comment:
Any reason one would be sparse and the other dense?

Reply:
check if concated is sparse, if so and if standardization is on, it will be densified in order to use standardization.